### PR TITLE
Add dependabot

### PR DESCRIPTION
**Pull request by SHAcollision**
Add monthly dependabot.

Skip `nx` dependencies. These need manual migration with `npx nx migrate latest` .

### CODE REVIEW

**Comment by MiguelMedeiros** [comment]
Hey @SHAcollision! 

For the Dependabot configuration in the PR, could you confirm which GitHub username(s) should be set as the reviewers and assignees? 

Should I add my username, yours, or someone else’s? I just want to make sure everything is set up correctly. Thanks! 👍

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      # - dependency-name: "nextjs-toploader"
+      # - dependency-name: "tailwindcss"
+      # - dependency-name: "tailwind-merge"
+      # # Optionally ignore Cypress if you prefer manual updates
+      # - dependency-name: "cypress"
+      # - dependency-name: "eslint-plugin-cypress"
+    reviewers:
+      - "your-github-username"
+    assignees:
+      - "your-github-username"
```
Comment (MiguelMedeiros) [inline .github/dependabot.yml]: Comment this out, so we don't get the alerts:

```suggestion
    # reviewers:
    #  - "your-github-username"
    # assignees:
    #  - "your-github-username"
```

**Comment by SHAcollision** [comment]
Looks good. Better if there is no one automatically assigned, could be annoying.

**Review by MiguelMedeiros — APPROVED** [review]
ACK

**Pull request merged**